app/views/manage_submissions/_submissions_table.html.erb

```diff
@@ -0,0 +1,71 @@
+<table class="usa-table usa-table--stacked-header usa-table--borderless width-full">
+    <thead>
+      <tr>
+        <th scope="col">Submission ID</th>
+        <th scope="col">Eligible for Evaluation</th>
+        <th scope="col">Selected to Advance</th>
+        <th scope="col">Assigned Evaluators</th>
+        <th scope="col">Average Score</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @submissions.each do |submission| %>
+          <tr>
+            <th data-label="Submission ID" scope="row">
+              <span class="mobile-lg:display-none">Submission ID </span><%= submission.id %>
+            </th>
+            <td data-label="Eligible for Evaluation">
+              <div class="display-flex flex-align-center">  
+                <% if eligible_for_evaluation?(submission) %>
+                    <%= image_tag(
+                        "images/usa-icons/verified.svg",
+                        class: "usa-icon--size-3",
+                        alt: ""
+                    )%>
+                    Eligible for Evaluation
+                <% else  %>
+                    <%= image_tag(
+                        "images/usa-icons/highlight_off.svg",
+                        class: "usa-icon--size-3",
+                        alt: ""
+                    )%>
+                    Not Eligible for Evaluation
+                <% end %>
+              </div>  
+            </td>
+            <td data-label="Selected to Advance">
+              <div class="display-flex flex-align-center">  
+                <% if selected_to_advance?(submission) %>
+                    <%= image_tag(
+                        "images/usa-icons/star.svg",
+                        class: "usa-icon--size-3 margin-right-1",
```

Comment:
sorry, I meant can we add `margin-right-1` too all the `image_tag` icons on this page, not just this one?